README.md

```diff
@@ -77,51 +96,72 @@ submodule-cloning techniques:
 ### 3. Run installation scripts
 
 > NOTE: It is safe to (re-)run all of the commands and scripts given below even
-if you already have the required packages installed.
+> if you already have the required packages installed.
 
 **Open a bash terminal/command window** and execute the following commands:
 
-1. <code>cd <i>\<path-to-this-repo></i></code> &nbsp;&nbsp;# change to
-   **root of this repo**
-1. `source ./tool/env-set.sh` &nbsp;&nbsp;#
-   initialize environment variables; install/use required Node & Ruby version
-1. `./tool/before-install.sh` &nbsp;&nbsp;#
-   install core set of required tools
-1. `./tool/install.sh` &nbsp;&nbsp;#
-   install everything else needed to build this site
+1. After you have cloned this repo, change to the _root of this repo_:
+
+   ```console
+   $ cd <PATH_TO_REPO>
+   ```
+1.  Run the `env-set.sh` script to initialize environment variables, and to install/use required Node & Ruby version:
+
+    ```console
+    $ source ./tool/env-set.sh
+    ```
+1.  Run `before-install.sh` to install the  core set of required tools:
+
+    ```console
+    $ ./tool/before-install.sh
+    ```
+1.  Run `install.sh` to install everything else needed to build this site:
+
+    ```console
+    $ ./tool/install.sh
+    ```
```

Comment:
```suggestion
    ```

You _may_ need to run `gem install bundler` to upgrade it to the latest version if you get errors such as `require: cannot load such file` later. Another command that seems to be useful when the usual installation process doesn't work is `bundle install --force`.
```

Reply:
I just had to install site-www and these things helped me.